types/tests/augmentation-test.ts

```diff
@@ -0,0 +1,15 @@
+import Banana from '../index';
+
+declare module '../banana-i18n' {
+	type Augementation = 'key1'|'key2';
```

Comment:
Perhaps `Augmentation` in place of `Augementation`? (4 other occurrences in this file)

Reply:
Done